styles.css

```diff
@@ -73,10 +87,204 @@ main {
 li {
   list-style-type: none;
   color: #505f79;
-  transition: 0.3s all ease-in-out;
 }
 
 li :hover {
-  transform: rotate(30deg);
   color: #6070ff;
 }
+
+/* Work-Station Section */
+.work-section {
```

Comment:
**\[OPTIONAL\]**
- [ ] There's a lot of space at the end of the work section. Kindly fix that to match that of the Figma design
- [ ] Also add some space between the see project buttons and the technologies.
![image](https://user-images.githubusercontent.com/29071984/170832136-2c75c265-9970-4094-8e39-294cad28c7c7.png)